showcases/shared/link.json

```
@@ -5,12 +5,14 @@
 			{
 				"name": "(Default) Adaptive",
 				"code": {
+					"html": "<a class=\"db-link\" href=\"#\" tabindex=\"0\" data-content=\"internal\">(Default) Adaptive<span aria-hidden=\"true\" class=\"db-icon\" data-icon=\"link\"></span></a>",
```

```suggestion
					"html": "<a class=\"db-link\" href=\"#\" tabindex=\"0\" data-content=\"internal\">(Default) Adaptive</a>",
```

We shouldn't use another unnecessary tag, but either `data-icon` attribute or the already existing `.db-link` CSS declarations for `data-content` attribute.